assets/offering/Placeholder.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="232" height="242" viewBox="0 0 232 242" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Awesome with SVGs! 💥 However, what we discussed before was if we should maybe inline them as react components instead of loading them as images. @Iamsamwhoami @hedvigoscar thoughts?

Reply:
Yes!

In fact lets just create the package I think, it feels significant enough.

Reply:
Agreed!